features/environment.py

```diff
@@ -24,9 +24,20 @@
 
 
 def before_all(context):
-    config = get_config(context.config.vendor)
 
-    context.config = config
+    plan = []
+    for feature in context._runner.features:
```

Comment:
Since right now we only need the count of features, I'd be more inclined to leave out this whole loop.